tests/test_admin/test_view.py

```diff
@@ -961,27 +961,27 @@ def _post_multi_enroll(
         if create_user:
             user = UserFactory(email=user_email)
 
-        for course_id in courses:
-            catalog_instance.get_course_run.return_value = courses[course_id]
-            mode = courses[course_id]['mode']
+        for key, course_id in courses.items():
```

Comment:
[style nit] The `course_id` seems more like the key and the value (I believe?) is content metadata. I don't think this would break anything but might be more clear to a reader.